simulator/src/sequential/JKflipFlop.js

```
@@ -46,6 +48,9 @@ export default class JKflipFlop extends CircuitElement {
      * if none of the predefined nodes have been deleted it isresolvable
      */
     isResolvable() {
+        if(this.reset.value == 1 && this.preset.value == 1 || this.J.value == 1 && this.K.value == 1){
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)